mslib/plugins/io/gpx.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+
+    mslib.plugins.io.csv
+    ~~~~~~~~~~~~~~~~~~~~
+
+    plugin for csv format flight track export
+
+    This file is part of mss.
+
+    :copyright: Copyright 2008-2014 Deutsches Zentrum fuer Luft- und Raumfahrt e.V.
+    :copyright: Copyright 2011-2014 Marc Rautenhaus (mr)
+    :copyright: Copyright 2016-2021 by the mss team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+
+import os
+from fs import open_fs
+import gpxpy
```

Comment:
This is a new dependency. Needs to be added to localbuild/meta.yaml
@ReimarBauer anywhere else?

Reply:
it is on conda-forge

gpxpy                          1.4.2            py_0  conda-forge 


please add it  at current state only to localbuild/meta.yaml

At the time this becomes released we add it to the feedstock